### PR TITLE
adds an option to scroll to the playing audio

### PR DESCRIPTION
**Pull request by sharabash**
@mmahalwy

### CODE REVIEW

File: src/components/Audioplayer/index.js [hunk 1]
```diff
@@ -165,6 +196,24 @@ export default class Audioplayer extends Component {
...
+    event.preventDefault();
+
+    const { shouldScroll } = this.props;
+    const currentAyah = this.getCurrent();
+    const ayahNum = currentAyah.replace( /^\d:/, '' );
+
+    if (!shouldScroll) { // we use the inverse (!) here because we're toggling, so false is true
+      if (scroller.get('ayah:'+ ayahNum).getBoundingClientRect().top < 0) // if the ayah is above our scroll offset
+        scroller.scrollTo('ayah:'+ ayahNum, null, null, -120);
+      else
```
Comment (mmahalwy) [inline src/components/Audioplayer/index.js]: please have the curlys for if else `{}` :)

File: src/redux/modules/audioplayer.js [hunk 2]
```diff
@@ -145,6 +152,12 @@ export function repeat() {
   };
 }
 
+export function scroll() {
```
Comment (mmahalwy) [inline src/redux/modules/audioplayer.js]: `toggleScroll`

File: src/redux/modules/audioplayer.js [hunk 3]
```diff
@@ -145,6 +152,12 @@ export function repeat() {
...
+  return {
+    type: SCROLL
```
Comment (mmahalwy) [inline src/redux/modules/audioplayer.js]: `TOGGLE_SCROLL` sounds more descriptive.

File: src/components/Audioplayer/index.js [hunk 4]
```diff
@@ -231,6 +280,24 @@ export default class Audioplayer extends Component {
...
+
+    return (
+      <Col xs={2} className="text-center pull-right">
+        <input type="checkbox" id="scroll" className={style.checkbox} />
+        <label
+          htmlFor="scroll"
+          className={`pointer ${style.buttons} ${shouldScroll ? style.scroll : ''}`}
+          onClick={this.scroll.bind(this)}
+        >
+          <i className="ss-icon ss-openbook" />
```
Comment (mmahalwy) [inline src/components/Audioplayer/index.js]: I feel like users might not understand what this means. We should probably use another icon or text. See https://github.com/quran/quran.com-frontend/blob/master/static/fonts/ss-standard/documentation.html

**Comment by mmahalwy** [comment]
just some comments and you're good to go! Nicely done :)

File: src/components/Audioplayer/index.js (hunk 4 above)
Comment (sharabash) [inline src/components/Audioplayer/index.js]: none of the icons look right; the closest one i think is ss-list. what's your opinion?

**Pull request merged**